src/Host/Client/Impl/Definitions/REvaluationResult.cs

```diff
@@ -31,6 +32,14 @@ public struct REvaluationResult {
         /// </remarks>
         public JToken Result { get; }
         /// <summary>
+        /// Result of evaluation for 'raw'
+        /// </summary>
+        /// <remarks>
+        /// Computed by serializing the immediate result of evaluation, as if by <c>serialize(x,NULL)</c>
```

Comment:
Is this comment still correct? Looking at the host code, it seems that it just handles `RAWSXP`, and doesn't try to serialize anything else?